test/e2e/batch_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestBatchCreatePods verifies there is no FD leak after batch creating Pods.
+func TestBatchCreatePods(t *testing.T) {
```

Comment:
do you think we should skip this test on Kind? Or does it work consistently despite the limited size of the Nodes and does it complete quickly enough?

Reply:
It works on kind (all kind tests before I re-pushed), the busybox pod should be very light weight. It took 40~50 seconds in my testbed, will check the time spent on kind after this round.

Reply:
thanks, that sounds good then, let's keep it for now

Reply:
It takes 40~50 seconds too on kind.
```
PASS: TestBatchCreatePods (44.54s)
PASS: TestBatchCreatePods (40.05s)
PASS: TestBatchCreatePods (44.41s)
```